extensions/wikia/UserProfilePageV3/filters/UserWikisFilterPrivateDecorator.class.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * Filters private wikis from user favourite wikis list
+ *
+ * Class UserWikisFilterPrivateDecorator
+ */
+class UserWikisFilterPrivateDecorator extends UserWikisFilterDecorator {
+	public function getFiltered() {
+		$filtered = $this->filter->getFiltered();
+
+		foreach( $filtered as $key => $wikiData ) {
+			$wikiId = (int) $wikiData['id'];
+
+			if( WikiFactory::getVarValueByName( 'wgIsPrivateWiki', $wikiId ) === true ) {
```

Comment:
Nice catch!
Overused Zen of Pyhon:
> [...]
> Explicit is better than implicit.
> [...]

But in this case it's definitely not required :)